apps/web/src/app/auth/verify-recover-delete-org.component.html

```diff
@@ -0,0 +1,34 @@
+<form #form (ngSubmit)="submit()" [appApiAction]="formPromise" class="container" ngNativeValidate>
+  <div class="row justify-content-md-center mt-5">
+    <div class="col-5">
+      <p class="lead text-center mb-4">{{ "deleteAccount" | i18n }}</p>
+      <div class="card">
+        <div class="card-body">
+          <app-callout type="warning">{{ "deleteAccountWarning" | i18n }}</app-callout>
+          <p class="text-center">
+            <strong>{{ name }}</strong>
+          </p>
+          <p>{{ "deleteRecoverConfirmDesc" | i18n }}</p>
```

Comment:
Feedback from product: 
>Add the following sentence at the end: User accounts will remain active after deletion but will no longer be associated to this organization.
```suggestion
          <p>{{ "deleteOrganizationConfirmDesc" | i18n }}</p>
          <p>{{ "deletingOrganizationActiveUserAccountsWarning" | i18n }}</p>
```